README.md

```diff
@@ -86,7 +86,7 @@ Fileset passed to every task has metadata `(:metadata (meta fileset)`. This meta
 ## Install
 
 ```clojure
-[perun "0.1.0-SNAPSHOT"]
+[perun "0.1.3-SNAPSHOT"]
 ```
```

Comment:
Since you are using bootlaces you can replace this with:
[](dependency)
```clojure
blabla
```
[](/dependency)
and the `build-jar` task will insert the coordinates of the latest build automatically :)